src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void markDone(){
+        this.isDone = true;
+    }
+
+    public void markUndone(){
+        this.isDone = false;
+    }
```

Comment:
Suggestion: can be combined into a method named setTaskStatus(boolean state) which sets the state of the task as complete (true) or not complete (false) depending on the value of state variable (this.isDone = state;).